test.html

```diff
@@ -26,7 +26,7 @@
   </p>
 
 
-  <form id="gform" method="POST" class="pure-form pure-form-stacked" action="https://script.google.com/macros/s/AKfycbyF7zsZXQUZLKBOaVKfsrzyr30FPHprgqd8flvC5WeuYQJnc5E/exec">
+  <form id="gform" method="POST" class="pure-form pure-form-stacked" data-email="contact.nelsonic+form.submit@gmail.com" action="https://script.google.com/macros/s/AKfycbyF7zsZXQUZLKBOaVKfsrzyr30FPHprgqd8flvC5WeuYQJnc5E/exec">
```

Comment:
@mckennapsean feel free to _assign_ the PR to me when you feel it's ready to ship. 👍